core/storage/classifier/gae_models.py

```diff
@@ -24,9 +24,14 @@
 import python_utils
 import utils
 
-(base_models,) = models.Registry.import_models([models.NAMES.base_model])
+from typing import Any, Dict, List, Optional, Text, Tuple, Union # isort:skip # pylint: disable=unused-import
 
-datastore_services = models.Registry.import_datastore_services()
+MYPY = False
+if MYPY:
+    from mypy_imports import * # pragma: no cover # pylint: disable=import-only-modules,wildcard-import,unused-wildcard-import
+else:
+    (base_models,) = models.Registry.import_models([models.NAMES.base_model])
+    datastore_services = models.Registry.import_datastore_services()
```

Comment:
Why are these imports inside an if block only in this file and not others?

Reply:
Removed. Made it by mistake.